Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1399,6 +1399,101 @@ As per `k8s Service <https://kubernetes.io/docs/concepts/services-networking/ser
 Cilium's eBPF kube-proxy replacement by default disallows access to a ClusterIP service from outside the cluster.
 This can be allowed by setting ``bpf.lbExternalClusterIP=true``.
 
+Observability
+*************
+
+You can trace socket LB related datapath events using Hubble and cilium monitor.
+
+Apply the following pod and service:
+
+.. code-block:: yaml
+
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: nginx
+      labels:
+        app: proxy
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:stable
+        ports:
+          - containerPort: 80
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: nginx-service
+    spec:
+      selector:
+        app: proxy
+      ports:
+      - port: 80
+
+Deploy a client pod to start traffic.
+
+.. code-block:: shell-session
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes-dns/dns-sw-app.yaml
```

Comment:
This one must be a `parsed-literal` for the substitution on `|SCM_WEB|` to happen.
```suggestion
.. parsed-literal::

    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes-dns/dns-sw-app.yaml
```